src/tabcenter.js

```diff
@@ -99,9 +99,22 @@ TabCenter.prototype = {
     for (let identity of identities) {
       const identityItem = document.createElement("div");
       identityItem.className = "newtab-menu-identity";
-      identityItem.addEventListener("mouseup", () => {
-        this.hideNewTabMenu();
-        browser.tabs.create({ cookieStoreId: identity.cookieStoreId });
+      identityItem.addEventListener("mouseup", async e => {
+        switch (e.which) {
+        case 1:
```

Comment:
Okay, maybe this can reduce the memory footprint for variable storage. I does currentIndex to clarify its semantics. This should be refactored as a function after merge (avoid it be a conflict or a larger pr).